batchflow/models/torch/base.py

```diff
@@ -974,6 +984,8 @@ def train(self, *args, feed_dict=None, fetches=None, use_lock=False, train_mode=
             splitted_inputs = [inputs]
             splitted_targets = [targets]
 
+        if use_lock:
+            self.train_lock.acquire()
```

Comment:
`model` must be always created with `lock`, independently of `use_lock` parameter

Reply:
done